lib/components/app/responsive-webapp.js

```diff
@@ -237,29 +235,19 @@ class RouterWrapperWithAuth0 extends Component {
               render={() => <WebappWithRouter {...this.props} />}
             />
             <Route
-              // This route lets new or existing users edit or set up their account.
-              path={'/account'}
-              component={routerProps => {
-                const props = this._combineProps(routerProps)
-                return <UserAccountScreen {...props} />
-              }}
+              path={`${TRIPS_PATH}/:id`}
+              component={SavedTripScreen}
             />
+            <Route exact path={ACCOUNT_PATH}>
```

Comment:
A change in one of the `Route` elements is causing these errors:
```log
checkPropTypes.js:20 Warning: Failed prop type: Invalid prop `path` of type `array` supplied to `Route`, expected `string`.
    in Route (created by RouterWrapperWithAuth0)
    in RouterWrapperWithAuth0 (created by ConnectFunction)
    in ConnectFunction (created by OtpRRExample)
    in OtpRRExample
    ...
checkPropTypes.js:20 Warning: Failed prop type: Invalid prop `component` of type `object` supplied to `Route`, expected `function`.
    in Route (created by RouterWrapperWithAuth0)
    in RouterWrapperWithAuth0 (created by ConnectFunction)
    in ConnectFunction (created by OtpRRExample)
    in OtpRRExample
    ...
```

Reply:
These look to be due to [bugs](https://github.com/ReactTraining/react-router/issues/6566#issuecomment-459371333) in react-router 4.3.1, which I downgraded to so that we could use with react-router-bootstrap. I'm not sure it's really worth worrying about, but I'll see if there's another version of react-router et al. that we can use.

Reply:
Fixed in bbb6987f3a529c8851d6d7e868fabade4632872d.